### ISSUE TITLE
Allow users to add alt text for images

### ISSUE THREAD

**Issue by jeremiak** [18F/federalist#256]
As a user, I want to be able to supply alt text when I put an image into my content.

**Comment by wslack** [comment]
Closing as the editor has been disabled.
